runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -235,14 +235,14 @@ public NettyClientRouter(String host, Integer port, Boolean tls,
     public NettyClientRouter(String host, Integer port, Boolean tls,
                              String keyStore, String ksPasswordFile, String trustStore,
                              String tsPasswordFile, Boolean saslPlainText, String usernameFile,
-                             String passwordFile, MetricRegistry metricRegistry) {
+                             String passwordFile, MetricRegistry metricRegistry, long responseTimeout) {
```

Comment:
Can we avoid adding more fields to pass information (the current amount is kind of crazy).
Make a ConnectionSettings (or similar class) to pass this data....

Reply:
Yea, its pretty bad. Will address #991 first.